src/Misc.php

```diff
@@ -199,17 +199,12 @@ public static function fix_protocol($url, $http = 1)
         return $url;
     }
 
+    /**
+     * @deprecated since SimplePie 1.7.1, use PHP native array_replace_recursive() instead.
+     */
     public static function array_merge_recursive($array1, $array2)
     {
-        foreach ($array2 as $key => $value) {
```

Comment:
For BC reasons I would leave the Misc::array_merge_recursive() method untouched. I think that it is enough if we mark it as deprecated and no longer use it internally.

Reply:
Done